### PR TITLE
MWRAPPER-53 cygwin path fix for class like for source

### PR DESCRIPTION
**Pull request by nhojpatrick**
PullRequest/Patch for https://issues.apache.org/jira/browse/MWRAPPER-53.

### CODE REVIEW

File: maven-wrapper-distribution/src/resources/mvnw [hunk 1]
```diff
@@ -259,20 +259,22 @@ else
...
-        if [ -e "$javaClass" ]; then
-            if [ ! -e "$BASE_DIR/.mvn/wrapper/MavenWrapperDownloader.class" ]; then
+        if [ -e "$javaSource" ]; then
+            if [ ! -e "$javaClass" ]; then
                 if [ "$MVNW_VERBOSE" = true ]; then
                   echo " - Compiling MavenWrapperDownloader.java ..."
                 fi
                 # Compiling the Java class
-                ("$JAVA_HOME/bin/javac" "$javaClass")
+                ("$JAVA_HOME/bin/javac" "javaSource")
```
Comment (sepe81) [inline maven-wrapper-distribution/src/resources/mvnw:275]:
```suggestion
                ("$JAVA_HOME/bin/javac" "$javaSource")
```

i guess it was not intentional to remove $

**Review by sepe81 — CHANGES_REQUESTED** [review]

File: maven-wrapper-distribution/src/resources/mvnw (hunk 1 above)
Comment (nhojpatrick) [reply maven-wrapper-distribution/src/resources/mvnw]: yep

**Comment by hboutemy** [comment]
nice, thank you

**Comment by jira-importer** [comment]
Resolve #194

**Pull request merged**